tests/testsuite/patch.rs

```diff
@@ -2693,6 +2693,8 @@ fn mismatched_version_with_prerelease() {
 candidate versions found which didn't match: 0.1.1-pre1
 location searched: crates.io index
 required by package `foo v0.1.0 [..]`
+if you are looking for the prerelease package it needs to be specified explicitly
+prerelease-deps = { version = "0.1.1-pre1" }
 perhaps a crate was updated and forgotten to be re-vendored?"#,
```

Comment:
nit: Should we indent the suggested version req both for "quoting" it and for setting it apart from the suggestion that may come after it?